modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandArgIterator.java

```diff
@@ -97,19 +97,52 @@ public String peekNextArg() {
         return peekedArg;
     }
 
+    /**
+     * @return Numeric value.
+     */
+    public long nextPositiveLongArg(String argName) {
+        long val = nextLongArg(argName);
+
+        if (val < 0)
```

Comment:
Actually, val is "non-negative" rather "positive"
Let's rename methods.

Reply:
Fixed